src/pages/signIn/SignIn.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import { useNavigate } from "react-router-dom"
+import * as S from "./style"
+
+export default function signIn() {
+  const navigate = useNavigate()
+  const [idInput, setIdInput] = React.useState("")
+  const [pwInput, setPwInput] = React.useState("")
+  const [formCheck, setFormCheck] = React.useState("")
```

Comment:
```suggestion
import React, { useState } from "react"
import { useNavigate } from "react-router-dom"
import * as S from "./style"

export default function signIn() {
  const navigate = useNavigate()
  const [idInput, setIdInput] = useState("")
  const [pwInput, setPwInput] = useState("")
  const [formCheck, setFormCheck] = useState("")
```

그리고 `useState()`는 직접 import해서 바로 사용하는게 더 좋아요!
자주 쓰이니까 계속 React로 접근해야하니 코드양이 제법 늘어날 거라.. ㅎㅎ

Reply:
확인했습니다!